<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-M0OJOK9" failures="1" tests="2" name="Regression 1" time="1.189" errors="0" timestamp="2023-08-30T21:38:42 ICT">
  <testcase classname="Test1" name="loginTest" time="0.47"/>
  <testcase classname="Test1" name="test2" time="0.022">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Test1.test2(Test1.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Regression 1 -->
